include/arch/arm/aarch64/timer.h

```diff
@@ -17,6 +17,9 @@ extern "C" {
 #endif
 
 #define ARM_ARCH_TIMER_IRQ	ARM_TIMER_VIRTUAL_IRQ
+#define ARM_ARCH_TIMER_PRIO	ARM_TIMER_VIRTUAL_PRIO
+#define ARM_ARCH_TIMER_FLAGS	ARM_TIMER_VIRTUAL_FLAGS
+
```

Comment:
This could be in `include/drivers/timer/arm_arch_timer.h` itself, otherwise one has to ensure including `arm_arch_timer.h` before including `timer.h` (via `arch/cpu.h`) - which is ensured currently.

Reply:
The choice of which type of timer instance to use and its associated routines should be some where other than
`include/drivers/timer/arm_arch_timer.h`  and  `timer.h` seems fine to me .

Reply:
I see that `timer.h` already includes `drivers/timer/arm_arch_timer.h.` Got confused as both are included in driver file.